src/components/button/template.njk

```diff
@@ -10,9 +21,14 @@
   {% endif %}
 {% endif %}
 
+{% if params.isStartButton %}
+  {% set iconMarkUp = '<svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" height="19px" width="30px"><path fill="currentColor" d="M7.5 0H0l10 10L0 20h7.5l10-10-10-10z"/></svg>' %}
```

Comment:
We should check if we need `focusable=false` on internet explorer here

Reply:
I've tested this in IE and it doesn't appear to be an issue. I think this might because the svg is already in a focusable element like `a` and `button`